lib/home/library/video_library.dart

```diff
@@ -90,15 +85,15 @@ class VideoCard extends StatelessWidget {
   }
 
   // Add a method to download the video
-  Future<void> downloadVideo(BuildContext context) async {
+  Future<void> downloadVideo() async {
     try {
       // Retrieve the download URL from Firebase Storage
       final storageReference = FirebaseStorage.instance.ref().child(path);
       final url = await storageReference.getDownloadURL();
 
       if (kIsWeb) {
         // Handle download for web environment
-        final html.AnchorElement anchorElement = html.AnchorElement(href: url)
+        html.AnchorElement(href: url)
```

Comment:
Since we aren't using `anchorElement` further down, we don't need to assign to a variable here.